api/src/main/scala/hmda/api/protocol/processing/ParserResultsProtocol.scala

```diff
@@ -0,0 +1,40 @@
+package hmda.api.protocol.processing
+
+import hmda.api.model.{ PaginationLinks, ParsingErrorSummary, WithPagination }
+import hmda.parser.fi.lar.LarParsingError
+import spray.json._
+
+trait ParserResultsProtocol extends DefaultJsonProtocol {
+
+  implicit val larParsingErrorFormat = jsonFormat2(LarParsingError.apply)
+  implicit val paginationLinksFormat = jsonFormat6(PaginationLinks.apply)
+
+  implicit object ParsingSummaryJsonFormat extends RootJsonFormat[ParsingErrorSummary] {
+    override def write(summary: ParsingErrorSummary): JsValue = {
+      JsObject(
+        "transmittalSheetErrors" -> summary.transmittalSheetErrors.toJson,
+        "larErrors" -> summary.larErrors.toJson,
+        "count" -> JsNumber(summary.count),
+        "total" -> JsNumber(summary.total),
+        "_links" -> summary.links.toJson
+      )
+    }
+
+    override def read(json: JsValue): ParsingErrorSummary = {
+      json.asJsObject.getFields("transmittalSheetErrors", "larErrors", "count", "total", "_links") match {
+        case Seq(JsArray(ts), JsArray(lar), JsNumber(count), JsNumber(tot), JsObject(links)) =>
```

Comment:
`count` is not used, so you can replace this with `JsNumber(_)`. The codebase is inconsistent regarding the usage of a placeholder in these scenarios, I have created [this](https://github.com/cfpb/hmda-platform/issues/823) ticket to address it.